dotnet/src/Skills/Skills.OpenAPI/JsonSerializerContexts.cs

```diff
@@ -13,6 +14,23 @@ namespace Microsoft.SemanticKernel.Text;
 [JsonSerializable(typeof(JsonObject))]
 [JsonSerializable(typeof(List<object>))]
 [JsonSerializable(typeof(string))]
+// Speculatively adding a bunch more types that SharpYaml can produce
```

Comment:
What do you think about just not using the source generator here? The method is marked `RequiresUnreferencedCode`. We could just leave the code alone.

Reply:
I don't have a strong opinion.  I would expect, though, that we'd prefer things happen to work more often than not even for things marked RequiresUnreferencedCode.

Reply:
My only concern is breaking someone using this who isn't trimming. I'll leave it up to you to decide if you think the risk there is worth it. I guess this library hasn't officially shipped, so we don't really have to worry about regressions.